index.html

```diff
@@ -35,5 +35,146 @@ <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
         <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
       </header>
     </section>
+    <section id="middle">
+      <h2>My Recent Works</h3>
+      <div class="container">
+        <article class="grid-item">
```

Comment:
- [ ] Good job so Far :clap: , however, please ensure that your class name: `grid-item` is referenced correctly in your `styles.css` file and kindly use `flex` properties to align the contents of your card items. as part of the requirement that says: `In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.` please resolve it. :slightly_smiling_face: